pycqed/measurement/waveform_control/pulse_library.py

```diff
@@ -204,12 +203,12 @@ def chan_wf(self, channel, tvals):
     def hashables(self, tstart, channel):
         if channel in [self.I_channel, self.Q_channel]:
             return super().hashables(tstart, channel)
-        if self.pulse_off:
-            return ['Offpulse', self.algorithm_time() - tstart, self.length]
+        hashlist = self.common_hashables(tstart, channel)
```

Comment:
Is this related to Cryoscope?
Same question for all the changes in this file.

Reply:
see my other response